driver/src/contracts/stablex_contract.rs

```diff
@@ -0,0 +1,152 @@
+#[cfg(test)]
+extern crate mock_it;
+
+use dfusion_core::models::{AccountState, Order};
+
+// TODO - uncomment when removing "unimplemented!()"
+//use web3::contract::Options;
+//use web3::futures::Future;
+use web3::types::{H160, U128};
+
+use crate::error::DriverError;
+
+use super::base_contract::BaseContract;
+
+use std::env;
+use std::fs;
+
+type Result<T> = std::result::Result<T, DriverError>;
+
+#[allow(dead_code)]
+struct StableXContractImpl {
+    base: BaseContract
+}
+
+#[allow(dead_code)]
+impl StableXContractImpl {
+    pub fn new() -> Result<Self> {
+        let contract_json = fs::read_to_string("dex-contracts/build/contracts/StablecoinConverter.json").unwrap();
+        let address = env::var("STABLEX_CONTRACT_ADDRESS").unwrap();
```

Comment:
Given that you return a result, you shouldn't need to use any unwraps here. In case any line fails, new should return the error flavor of Result.

Reply:
Ok. I was not aware of that.

Reply:
I am pretty sure these need to be unwrapped in order for the next line to work.

```
BaseContract::new(address, contract_json).unwrap()
```